lib/error.js

```diff
@@ -276,4 +275,15 @@ AppError.staleAuthAt = function staleAuthAt(authAt) {
   });
 };
 
+AppError.notConfidentialClient = function notConfidential(clientId) {
+  return new AppError({
+    code: 400,
+    error: 'Bad Request',
+    errno: 120,
```

Comment:
Can you document this error value [in api.md](https://github.com/mozilla/fxa-oauth-server/blob/master/docs/api.md)? This error will need to be added to the content-server's [oauth-errors file](https://github.com/mozilla/fxa-content-server/blob/29c7fb8e473d53d793d85766ccaaaae89dd2a1ab/app/scripts/lib/oauth-errors.js#L90) too. It looks like someone forgot errno 119 so I [opened a bug](https://github.com/mozilla/fxa-content-server/issues/6358).